### DISCUSSION TITLE
[feat, flutter/dart] check if value in array for streams (`.overlap()` for streams)

### DISCUSSION THREAD

**Discussion by opxdelwin** [supabase/supabase#27674] [Feature Requests]
Currently `.overlap()` can only be used with `.select()` and not with `.stream()`. Our app needs to listen to the db realtime, and filter out rows whose column contain a value provided.

Eg.
Listen to 
```
Groups(
  members text[]   //or uuid[]
) 
```
such that the stream catches only rows with member containing 'abc'

**Comment by GaryAustin1** [comment]
My understanding is streams uses a basic realtime call underneath.  Realtime only support 7 or so basic calls with .in() being the most complex one.  https://supabase.com/docs/guides/realtime/postgres-changes?queryGroups=database-method&database-method=sql&queryGroups=language&language=js#available-filters

So nothing the REST clients can do about it unless realtime adds new filter types.

**Comment by opxdelwin** [comment]
so for the time being, I can only run this filer on rest client, without realtime changes?

**Reply by GaryAustin1** [reply]
Unless you can use RLS to also help with filtering (like based on the user id).  Otherwise you would have to post process or send more of a signal there was a change and then do a fetch back to see what it was.

**Reply by opxdelwin** [reply]
For the time being, I put it as refreshable, so for new data, user would have to refresh...
